src/app/components/profile/pages/annotations/annotations.component.html

```diff
@@ -22,7 +22,9 @@ <h1>
   >
     <ngx-datatable-column name="Site" [sortable]="false">
       <ng-template let-value="value" ngx-datatable-cell-template>
-        {{ value.audioRecording?.site?.name || "(loading)" }}
+        <span
+          [innerHTML]="value.audioRecording?.site?.name | spinner: 'sm'"
```

Comment:
when you get to the point of modifying html, it is probably time for a component?